.github/ISSUE_TEMPLATE/imperfection_report.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Hmm not sure about this, I'm thinking it might be confusing for users, the more options, the more thought that will have to go into this (i.e. more friction), and therefore reduced number of reports.

Reply:
I agree that too many options is a bad thing as well, but I've found that some issues just can't be boxed into either "fix bug" or "build feature", because they are about improving things that are working fine, just could be made better – _perfected_. That may for instance include refactors or UI cleanup.

Reply:
Thanks for addressing the feedback! This would be my only outstanding issue with the PR.

I still have this feeling that this will detract more than help. If a user wants to report an imperfection issue, they'll choose one of the categories and we can tag it later. My concern is that deciding between enhancement and imperfection is not clear-cut (as for instance deciding between enhancement and bug report). Basically I'm trying to follow a _Don't make me think_ principle.

Having said that, if you still think it's the right call feel free to merge as is, worst case scenario we can remove it in a few weeks.

Reply:
It's not actually supposed to be a choice between "enhancement" and "imperfection", but between "_feature_" or "imperfection". So it's more clear-cut (though I agree maybe not 100%) – both are enhancements, but one is about adding a feature to PostHog, the other about improving an existing feature. I feel like at times with the current simplistic distinction between "feature" and "bug" we _are_ making you think, because for example a refactor idea isn't an actual feature, so does "Feature request" apply to it? – it definitely can be considered perfecting the software though.

Reply:
Hm, I'm not actually sold on the specific wording "imperfection" though… I'll give this some more thought later, but for now merging without Imperfection report.